structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,201 @@
+
+The veto process may be started on behalf of another Manager or Collaborator. Additionally, the list of voters on that veto may be kept secret. This is intended to avoid exposing those that supported the veto (or that didn't) to negative reactions or bullying by other Program members.
+
+## Process: Adding a community
+
+Any Minecraft-related community may apply to join the Program --- but, given that the Program itself largely exists on Discord, it's easier for us to work with Discord-oriented communities.
+
+### Evaluation criteria
+
+Communities that apply to the Program are evaluated based on a number of criteria, which are examined and discussed by the existing Program members during the application. These criteria include, but are not limited to:
+
+* The main focus of the community
+* Why the community wishes to join the Program
+* How competently the community has been set up
+* The community's rules and approaches to moderation
+* Who is part of the community's staff team
+* Whether the community has any known, relevant history
+
+### Application process
+
+The application process is defined as follows:
+
+1. An administrator or moderator from the community that wishes to apply (henceforth, "the new community") must get in touch with a Manager from the Program to start the process. The following information is collected by the manager:
+    * The name and main focus of the community
+    * Why the community wishes to join the Program
+    * What the community rules are (or where to find them)
+    * A full list of staff members and their roles
+    * A method of accessing the community directly (for example, a Discord server invitation)
+2. Once the above information has been provided, a Manager informs voting-eligible Program members on the Discord server, and opens a discussion thread to figure out how the Program members feel about the new community, and to discuss anything relevant to the application
+3. A Discord channel is set up to allow the administrators and moderators from the new community (who are to be invited and given the Invitee role) to talk with existing Program members regarding their application
+4. Once the voting-eligible Program members have informally agreed that enough information has been gathered, the vote may be created (with options to approve or deny the vote) --- it must be done in a channel the Invitees don't have access to, and follow the voting process outlined earlier in this document
+5. Once the vote has concluded, the discussion channel must be archived and kept read-only, so it can be reviewed later on if needed
+
+### Interpreting the vote
+
+**The vote has passed** when more than 75% of the votes cast at the time of closing are positive. Internal discussions can be removed as needed before the new community is given access to the rest of the server. The new community should also be added to any relevant listings.
```

Comment:
Should there be a clause to additionally and explicitly require some % of the membership to have cast a vote? Or are we banking on the fact that within the mandated timeframe, communities will have cast a vote before closing?

Reply:
I think ten days is more than enough time to ensure that communities have voted. It's likely safe to assume they don't want to vote on the poll if they haven't by this time.